docs/platforms/javascript/guides/node/index.mdx

```diff
@@ -37,16 +37,28 @@ Sentry should be initialized as early in your app as possible. It is essential t
 
 Once this is done, Sentry's Node SDK captures unhandled exceptions as well as tracing data for your application.
 
+Therefore, we recommend to create a file named `instrumentation.js` that imports and initializes Sentry.
```

Comment:
Nitpick regarding line 36 and 46 in this file: Can you change the dash to an [em dash](https://www.thepunctuationguide.com/em-dash.html)?